{{cookiecutter.project_slug}}/CHEATSHEET.rst

```diff
@@ -56,13 +56,13 @@ To run tests
 * Actually run the tests: ``pytest {{cookiecutter.package_name}}``
 
 
-To run yapf
+To run black
 -----------
```

Comment:
I think you might need to extend the underline to match the length of the previous line? ReST is weird.